ci/jenkins/test-vmc.sh

```diff
@@ -315,7 +321,9 @@ function deliver_antrea {
     sed -i -e "s/idleFlowExportTimeout: \"15s\"/idleFlowExportTimeout: \"1s\"/g" $GIT_CHECKOUT_DIR/build/yamls/$antrea_yml
 
     wget -c https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/flow-aggregator.yml -O ${GIT_CHECKOUT_DIR}/build/yamls/flow-aggregator.yml
-    sed -i '215s/.*/      enable: true/' ${GIT_CHECKOUT_DIR}/build/yamls/flow-aggregator.yml
+
+    # sed -i '215s/.*/      enable: true/' ${GIT_CHECKOUT_DIR}/build/yamls/flow-aggregator.yml
+    perl -0777 -i.original -pe 's/    clickHouse:\n      # Enable is the switch to enable exporting flow records to ClickHouse.\n      #enable: false/    clickHouse:\n      # Enable is the switch to enable exporting flow records to ClickHouse.\n      enable: true/igs' $GIT_CHECKOUT_DIR/build/yamls/flow-aggregator.yml
```

Comment:
Recommend to use yq instead of regular expression to make it easier for maintenance.